### ISSUE TITLE
Cannot download the precpomputed files 

### ISSUE THREAD

**Issue by yinfeiy-g** [facebookresearch/cc_net#7]
Hi 

I am trying to reproduce the results from your paper. However, after downloading the common crawl data from aws, the access to the precomputed files seems failed. 

Did you changed the location of precomputed file ?

The error messages are like below:

```
/.local/lib/python3.7/site-packages/cc_net/jsonql.py:1141: 
UserWarning: Swallowed error HTTPSConnectionPool(host='dl.fbaipublicfiles.com', port=443): Max retries exceeded with url: 
/cc_net/2019-09/en_head_0017.json.gz (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7ff4d87913d0>: 
Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')) while downloading https://dl.fbaipublicfiles.com/cc_net/2019-09/en_head_0017.json.gz (2 out of 3)
```

**Comment by gwenzek** [comment]
Can't reproduce. I thinks it's a temporary issue ?
I don't think this AWS bucket has very high quotas.

**Comment by yinfeiy-g** [comment]
ok, reducing the parallelism and it is working now. Thanks

For reproduce the results, how much disk space are required ?

**Comment by yinfeiy-g** [comment]
Ping.

I have a machine with 1000G disk space but I still ran into out of disk space issue. Wondering how much is needed here.

**Comment by gwenzek** [comment]
Hi, you will need something like 10TB if I remember correctly with the current version of the code.
I'm currently working on a new release of code and data that will limit the disk usage to what you actually extract, 2TB for all languages.

**Comment by leogao2** [comment]
Is there a more exact estimate of the current disk space requirements?

**Comment by gwenzek** [comment]
I've a new version in the dev branch that will do streaming and cut the diskspace requirements to only what you keep so ~2TB. I'll update the issue once I merge this in master.

**Comment by gwenzek** [comment]
This has been merged in #13 .
